debezium-server-core/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+debezium.api.enabled=false
```

Comment:
Please see https://github.com/debezium/debezium-server/blob/main/debezium-server-dist/src/main/resources/distro/conf/application.properties.example

Reply:
Just for context. So, the problem is that by this file needs to be renamed and modified -- which means it can't really be  used to set defaults as user may easily omit them. 

However we do place this file on class path and thus run into a problem with classloader order -- although I believe that the `application.properties` inside the jar will be loaded first (and thus the one we add in startup script would be able to override the values inside it) it is not something we should be relying on. 

The main issue is that we shouldn't be adding the configuration file on cp in the first place. Quarkus (or rather smallrye config) will also automatically recognise `config/application.properties` and give it a higher precedence than cp config by default. However, to make things more complicated -- we don't just add the config file on cp, but rather the entire `conf` directory since while `application.properties` doesn't have to be on cp, things like `conf/cassandra/driver.conf` do.

Reply:
The discussed idea of not changing this at the moment and just relying on default in code will also not be feasible -- while it would generally work for this specific property, it pretty much means that we would not be able to do partial configuration mapping to a java class (which is cleaner than having bunch of `@ConfigProperty` randomly in the code) as that needs unknown property validation to be disabled (see the next line `smallrye.config.mapping.validate-unknown=false`). 

Thus as annoying as the change is, we do need to sort this out

Reply:
Opened #120 which needs to be merged prior to this